android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -1004,12 +1002,9 @@ public void onClick(View v) {
                         }
                     }));
         } else {
-            DecimalFormat df = new DecimalFormat("#.###");
-            df.setRoundingMode(RoundingMode.CEILING);
-            df.setMinimumFractionDigits(3);
             TextView estimatedRange = mPopupView.findViewById(R.id.estimated_range);
-            String minValue = df.format(minEarningsThisMonth);
-            String maxValue = df.format(maxEarningsThisMonth);
+            String minValue = BraveRewardsHelper.getFormattedAmount(0.008963);
```

Comment:
Seems like there are debug values still in here?

Reply:
oops sorry i am updating it.